magic6/magic6.cpp

```diff
@@ -0,0 +1,83 @@
+#include <fstream>
+
+using namespace std;
+
+ifstream input("magic6.in");
+ofstream output("magic6.out");
+
+char character;
+int n;
+int frequencyVector[10] = {};
+
+void verifyIfItsIntegerOrChar()
```

Comment:
now you have integer or char. What will you do if a new data type would appear in the context? You should change the name, maybe processCharacters(), separateCharacters(), etc